Author: José Manuel Santamaría Lema <panfaust@gamil.com>
Forwarded: no
Description: Quick patch remove the tab for showing changelog because it doesn't
 work yet.
---
 libmuon/DetailsWidget.cpp |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/muon/DetailsWidget.cpp
+++ b/muon/DetailsWidget.cpp
@@ -50,7 +50,7 @@ DetailsWidget::DetailsWidget(QWidget *pa
     m_detailsTabs.append(new DependsTab(this));
     m_detailsTabs.append(new InstalledFilesTab(0));
     m_detailsTabs.append(new VersionTab(0));
-    m_detailsTabs.append(new ChangelogTab(this));
+    //m_detailsTabs.append(new ChangelogTab(this));
 
     // Hide until a package is clicked
     hide();
